### DISCUSSION TITLE
Diskussion über das Ergebnis (Teil 5)

### DISCUSSION THREAD

**Discussion by DeepRamful** [DeepRamful/Gruppenarbeit-GitHub#36] [General]
Was können wir aus den Ergebnissen schließen?

**Comment by vafkoo** [comment]
Funktion D die ich geschrieben habe, ausgibt die Werte von Korrelation in einer Liste und es kann nützlich sein, trotzdem es ist schlecht darstellbar in eine Normale Korrelation-Plot, weil die Mathe_LK Werte sind nur 0 und 1.

**Reply by Jungangyoung** [reply]
Ja, denn 0 zugegebene Wert haben eine große Wirkungen an Zahlen sind Grafiken damit sehr schlecht visualisierbar.
aber 0 und 1 selbst sind literal sinnvoll zuzuordnen, wenn man Ja-Nein trifft.

**Comment by QuangHuy14899** [comment]
Von dem Ergebniss kann man sehen, dass sich nicht so viele Studierenden für Mathematik interessieren. Obschon das Interessenniveau von 6 höchste Häufigkeit aufweist, befindet sich die Mehrheit der Studierenden auf den Interessenniveaus von 1 bis 4. Von den Interessenniveaus 5 bis 7 hat nur das Interessenniveau von 6 die Häufigkeit, die größer als 10 ist, während von 1 bis 4 alle Interessenniveaus die Häufigkeit haben, die größer als 10 ist.

**Reply by DeepRamful** [reply]
Ja, gut beobachtet. Du kannst dir einen besseren Überblick verschaffen, indem du die Interpretation(e) analysierst, bei der wir die Stufen 1-7 durch hoch, mittel, niedrig ersetzt haben. Die Funktion zur Umwandlung der Werte in hoch, mittel und niedrig vereinfacht unsere Analyse erheblich.

**Reply by QuangHuy14899** [reply]
Ja stimmt, die Anzahl der Studierenden mit ''hoch'' Interesse ist am kleinsten. Trotzdem ist die nur ein bisschen kleiner als die Anzahl der Studierenden mit ''niedrig'' Interesse

**Comment by JustNastyya** [comment]
Als Ergebnis würde ich sagen, dass es fast keine Korrelation zwischen Studienfach und Interesse von Studierenden an solchen Disziplinen, wie Mathematik und Programmierung gibt. Doch im Durchschnitt gibt es viel mehr Menschen, die hohe Interesse an Mathematik zeigen, als an Programmierung, obwohl die meisten Studierenden mittlere Interesse an beiden Fächern zeigen.

**Reply by DeepRamful** [reply]
Ja, du hast Recht, denn wir haben bei der Erstellung unseres Datensatzes keine besondere Beziehung berücksichtigt. Völlig zufällige Datensätze führen zu keiner Korrelation.

**Reply by vafkoo** [reply]
Ja, man kann die Zusammenhang klar nicht sehen

**Comment by DeepRamful** [comment]
Bei der Erstellung des Datensatzes haben wir insbesondere bei Interesse_an_Mathematik und Interesse_an_Programmieren keine Korrelation berücksichtigt, da wir den Datensatz sehr zufällig gestalten wollten. Wir hatten einige Gedanken, zum Beispiel, dass wir eine Beziehung herstellen sollten, dass Mathematik- und Statistikstudenten ein höheres Interesse an Mathematik haben sollten und dasselbe für Informatikstudenten und Programmieren. Aber in Wirklichkeit ist es nicht so. Also haben wir uns für das Zufallsprinzip entschieden. Das Gleiche gilt für die Beziehung zwischen MatheLk und dem Kurs, den die Studenten besuchen.

Deshalb können wir aus den Korrelationen, die wir berechnet haben, sehen, dass es keine Korrelation (siehe Interpretation c und d) oder nur eine sehr geringe Korrelation zwischen diesen Variablen gibt, wenn wir Zufallszahlen zum Füllen unserer Datensätze verwenden.

**Reply by QuangHuy14899** [reply]
Sollte man diese Korrelation anwenden ? z.b in Datenforschung

**Reply by JustNastyya** [reply]
Genau. Die Algorithmen, mithilfe von denen die Datensätze erstellt wurden, waren für gleichverteilte Zufallzahlen. Das heißt, das alle mögliche Merkmalausprägungen ungefähr gleiche Häufigkeite haben. Deshalb haben wir so ein Ergebniss bekommen

**Reply by DeepRamful** [reply]
> Sollte man diese Korrelation anwenden ? z.b in Datenforschung

Ich verstehe nicht ganz, was du meinst, aber ich glaube nicht, dass wir das Ergebnis nutzen können, um weitere Untersuchungen dazu durchzuführen. Wie ich bereits sagte, handelt es sich nicht um Daten aus der realen Welt.

**Reply by JustNastyya** [reply]
Meinst du gleich verteilte Zufallszahlen? Kurz gesagt, die Algorithmen, die R benutzt, um Zufallszahlen zu berechnen, funktionieren so, dass die Zahlen gleich... naja, gleichverteilt sind. und deshalb sind unsere Datensätze auch.. gleichverteilt

**Reply by QuangHuy14899** [reply]
> > Sollte man diese Korrelation anwenden ? z.b in Datenforschung
> 
> Ich verstehe nicht ganz, was du meinst, aber ich glaube nicht, dass wir das Ergebnis nutzen können, um weitere Untersuchungen dazu durchzuführen. Wie ich bereits sagte, handelt es sich nicht um Daten aus der realen Welt.

genau du hast die richtige Antwort gegeben, nach der ich gefragt hatte.

**Comment by DeepRamful** [comment]
Hätten wir also mit echten Daten gearbeitet, die beispielsweise an der Universität erhoben wurden, hätten wir eine bessere Korrelation zwischen dem Interesse an Mathematik und Programmierung und der Wahl des Studiengangs feststellen können, den die Schüler belegen. Normalerweise folgen einige ihrem Interesse, deshalb denke ich, dass wir eine bessere Korrelation sehen werden. 
Andererseits werden die Daten keine perfekte Korrelation zeigen, weil manche ein Fach vielleicht auch nicht mögen, aber trotzdem ein Fach wählen, vielleicht auf Druck der Eltern...
Ein weiteres Problem bei echten Daten sind Ausreißer oder fehlende Daten.
In unserem Datensatz sind wir nicht darauf gestoßen, aber wenn wir mit realen Daten arbeiten, müssen wir die Daten bereinigen und so weiter. (Nicht jeder Schüler gibt in Umfragen wahrheitsgemäße Daten an, oder?)

Insgesamt gab dieses Github-Projekt einen guten Vorgeschmack auf das, was wir später als Data Science/Statistik-Studenten tun werden. Das Verstehen der Daten, die Analyse, die Visualisierung und die Diskussion. Und ja, die Gruppenarbeit war auch eine schöne Erfahrung.

**Reply by JustNastyya** [reply]
Hier kann ich nur zustimmen, Deep. Interessante Erfahrung mit so einem Projekt in der Gruppe ist genau das, was wir alle brauchen
